src/streamlink/plugins/invintus.py

```diff
@@ -0,0 +1,50 @@
+import re
+import json
+try:
+    from urllib.parse import urlparse, parse_qs
+except ImportError:
+    from urlparse import urlparse, parse_qs
+
+from streamlink.plugin import Plugin
+from streamlink.stream import HLSStream
+
+
+class InvintusMedia(Plugin):
+    url_re = re.compile(r"https?://player\.invintus\.com/\?clientID=\d+&eventID=\d+")
+    WSC_API_KEY = "7WhiEBzijpritypp8bqcU7pfU9uicDR"  # hard coded in the middle of https://player.invintus.com/app.js
+    API_URL = "https://api.v3.invintusmedia.com/v2/Event/getDetailed"
+
+    @classmethod
+    def can_handle_url(cls, url):
+        return cls.url_re.match(url) is not None
+
+    def _get_streams(self):
+        query = parse_qs(urlparse(self.url).query)
```

Comment:
Why urlparse if you're defining a strict regex anyway? Just set regex capture groups